assessment.rb

```diff
@@ -1,12 +1,67 @@
 require 'csv'
 # Assessment answers to:
 
+#ran: rails g model Licensee
+#thenran: rake db:migrate
+
 # question 2:
+Person.create(surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63,weight: 132, dob: '1943-09-26',eye_color: 'Brown', hair_color: 'Black')
 
 # question 3:
 
+Licensees.transaction do
+          CSV.foreach(Rails.root + 'data/licensees.csv' headers: true) do | Licensee_row |
+
+            Licensee = Licensee_row.to_hash
+            next if Licensee.exists? Licensee
+
+            Licensee.create!Licensee
+
+          end #of next if  DO I need this? (I think maybe not)
```

Comment:
No, in the case of an inline `if`, there's no need for an `end`. 
`next if Licensee.exists? licensee`
translates exactly to
`if Licensee.exists? licensee
   next
end`